assets/js/builder/Views/SettingsFields.js

```diff
@@ -352,6 +353,10 @@ define( [], function() {
 					defaults.input_type  = 'url';
 				break;
 
+				case 'disabled':
```

Comment:
Also, instead of disabled, can't we have an "upsell" type here? makes more sense.
"disabled" is like a feature of an upsell.